test/k8sT/manifests/1.6/demo_ds.yaml

```diff
@@ -22,31 +22,45 @@ spec:
         key: node.cloudprovider.kubernetes.io/uninitialized
         value: "true"
 ---
-apiVersion: v1
-kind: Service
-metadata:
-  name: testds-service
-spec:
-  ports:
-  - port: 80
-  selector:
-    zgroup: testDS
-
----
 apiVersion: extensions/v1beta1
-kind: Deployment
```

Comment:
Does it need to be a DaemonSet?

Reply:
I want to make sure that each pod is in each node, so tunnel connectivity can be tested.